packages/react/src/River/RiverBreakout.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {Ref, forwardRef} from 'react'
+import clsx from 'clsx'
+import {RiverProps} from '.'
+import {useAnimation} from '../animation'
+import styles from './River.module.css'
```

Comment:
Given the this is a separate component now, can we make this a separate stylesheet called RiverBreakout.module.css?

Reply:
I pushed a separate branch where I attempted this if you want to take a look: `mp/river-breakout-separate-css`

Reply:
That branch looked great 👍, do you want to merge those changes in here? Or was it causing issues with the css modules? Happy to pair if that helps!

Reply:
It was causing issues with the CSS modules. A pairing session would be greatly appreciated.

Thanks @rezrah !